### PR TITLE
STM32 CAN Error Handling

### PR DESCRIPTION
**Pull request by bakerstu**

### CODE REVIEW

File: src/freertos_drivers/st/Stm32Can.cxx [hunk 1]
```diff
@@ -123,6 +130,19 @@ Stm32Can::Stm32Can(const char *name)
 #endif
```
Comment (balazsracz) [inline src/freertos_drivers/st/Stm32Can.cxx:130]: please set the interrupt priority on the third interrrupt as well.

**Review by balazsracz — APPROVED** [review]

File: src/freertos_drivers/st/Stm32Can.cxx [hunk 2]
```diff
@@ -413,9 +417,48 @@ void Stm32Can::tx_interrupt_handler()
...
 
+/*
+ * Stm32Can::sce_interrupt_handler()
+ */
+void Stm32Can::sce_interrupt_handler()
+{
+    if (CAN->MSR & CAN_MSR_ERRI)
+    {
+        /* error interrupt has occured */
+        CAN->MSR |= CAN_MSR_ERRI; // clear flag
```
[the remark this answers is not in the document]
Comment (balazsracz) [reply src/freertos_drivers/st/Stm32Can.cxx:432]: apparently this flag is "rc_w1" which means it is cleared by setting it to 1, as per the part documentation.

[the remark this answers is not in the document]
Comment (bakerstu) [reply src/freertos_drivers/st/Stm32Can.cxx:432]: This is actually correct. It is a "set to clear" register.

File: src/freertos_drivers/st/Stm32Can.cxx (hunk 1 above)
Comment (bakerstu) [reply src/freertos_drivers/st/Stm32Can.cxx:132]: Fixed.

**Pull request merged**